### ISSUE TITLE
Open-source all CI/CD scripts and configuration that runs in Jenkins   

### ISSUE THREAD

**Issue by dblock** [opensearch-project/opensearch-build#104]
Everything in GHA is open-source and in repos. The Jenkins scripts and configuration should follow suit. Anyone should be able to reproduce the OpenSearch CI/CD infrastructure after #103 and importing scripts and configuration that would be OSS.

**Comment by dblock** [comment]
This ticket was about open-sourcing, not just adding source control. How about we rename this issue into making this repo public?

**Comment by Poojita-Raj** [comment]
Sure, @dblock. Should I rename all the PRs as "Open-sourcing CI/CD scripts and config for XYZ Jenkins job" instead?

**Comment by dblock** [comment]
> Sure, @dblock. Should I rename all the PRs as "Open-sourcing CI/CD scripts and config for XYZ Jenkins job" instead?

Nah, I'll just open another issue to make this repo public (which needs to go through the usual and tedious process of its own).

**Comment by peterzhuamazon** [comment]
Can you double check this @peternied since you are working on the Jenkins Open Source.
Thanks.

**Comment by peternied** [comment]
Moving to the CI repo, will update this item as the migration effort occurs. Meta issue for tracking is https://github.com/opensearch-project/opensearch-ci/issues/2
